src/select/typeahead/TypeaheadSelect.tsx

```diff
@@ -71,7 +71,8 @@ function TypeaheadSelect({
     Boolean(selectedOptionLimit && selectedOptions.length >= selectedOptionLimit - 1);
   const typeaheadClassName = classNames("typeahead-select__input", {
     "typeahead-select__input--is-dropdown-menu-open": isMenuOpen,
-    "typeahead-select__input--can-select-multiple": canSelectMultiple
+    "typeahead-select__input--can-select-multiple": canSelectMultiple,
+    "typeahead-select--has-selected-options": Boolean(selectedOptions.length)
```

Comment:
```suggestion
    "typeahead-select__input--can-select-multiple": canSelectMultiple,
    "typeahead-select__input--has-selected-options": Boolean(selectedOptions.length)
```
I think it should start with `typeahead-select__input...` because the classname is passed to the input

Reply:
I am sorry, I think I wasn't aware of the intention of the change the first time.
I guess `typeahead-select--has-selected-options` class name was correct, but it should be added to `typeaheadSelectClassName`, not to the input class name, this way it will be applied to the top container, not just the input. This way it will give more control for the styling

```tsx
  const typeaheadSelectClassName = classNames(
    "typeahead-select__dropdown",
    "typeahead-select--has-selected-options",
    customClassName
  );
```

Reply:
Yea this should've been added to the container